src/lib/DALMeteorMethods/UserMethods.js

```diff
@@ -0,0 +1,17 @@
+/*eslint-disable*/
+Users = new Mongo.Collection.get("users");
+
+if(Meteor.isServer){
+    Meteor.publish("users", function(){
+        return Users.find({_id:this.userId});
```

Comment:
This actually isn't even needed, as info for current user is automatically published to the client by accounts package